ndc-reference/bin/reference/main.rs

```diff
@@ -143,11 +148,43 @@ async fn main() {
         ))
         .with_state(app_state);
 
-    // run it with hyper on localhost:8100
-    axum::Server::bind(&"0.0.0.0:8100".parse().unwrap())
-        .serve(app.into_make_service())
-        .await
-        .unwrap();
+    // start the server on localhost:<PORT>
+    let port = env::var("HASURA_CONNECTOR_PORT")
```

Comment:
Are these changes necessary? There's no requirement for the reference agent to implement the deployment spec.

Reply:
I know it seems pedantic, but I'd really like the reference agent to be as minimal as possible. Also, I'd like it to be an example of what is _not_ required in order to meet the spec.

Reply:
We use it for testing in the engine, so it's very useful to be able to run it on a given port. Right now we do this inside Docker but we'd like to make it possible to run tests without relying on Docker for our own code.